core/templates/domain/exploration/subtitled-html.model.ts

```diff
@@ -27,8 +27,8 @@ export class SubtitledHtml {
   // A null content_id indicates that the SubtitledHtml has been created
   // but not saved. Before the SubtitledHtml object is saved into a State,
   // the content_id should be set to a string.
```

Comment:
```suggestion
  // A null 'content_id' indicates that the 'SubtitledHtml' has been created
  // but not saved. Before the 'SubtitledHtml' object is saved into a State,
  // the 'content_id' should be set to a string.
```

Reply:
and also by `set to a string`, what exactly do you mean?

Reply:
Actually a legacy comment,I have not added this .See this https://github.com/oppia/oppia/pull/9919/files#r459343189 , the author then removed ` | null` as a type because it wasn't used anywhere in the frontend . But with stricter type checks there is a need to add `| null` because `schema-default-value.service.ts` uses this property with a null value.

Reply:
hmm, you may need to explain why it is doing that